src/redux/category/categorySplice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: 'Under Construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'Categories',
+  initialState,
+  reducers: {},
+});
+
+export default categoriesSlice.reducers;
```

Comment:
- [ ] Good job you did her 👏 . However based on project requirements, you're to define an action type for checking the status. Then export the `checkStatus` as an action

## Possible implementation:
```
reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
```
## Requirement 

![image](https://user-images.githubusercontent.com/50990925/236953213-5ce8990a-be68-4c8a-b5b7-d817b0b540dd.png)